app/components/recirculation-prefooter.js

```diff
@@ -24,6 +24,40 @@ export default Component.extend(
 		isVisible: false,
 		liftigniter: inject.service(),
 		i18n: inject.service(),
+		hasNoLiftigniterSponsoredItem: true,
+		isInRightCountry: false,
+		shouldShowPlista: computed.and('hasNoLiftigniterSponsoredItem', 'isInRightCountry'),
+		plistaSponsoredContent: [],
+		fetchPlista() {
+			const plistaURL = 'http://farm.plista.com/recommendation/?publickey=845c651d11cf72a0f766713f&widgetname=api' +
+							'&count=1&adcount=1&image[width]=583&image[height]=328';
+			return fetch(plistaURL)
+				.then(response => {
+					return response.json();
+				})
+				.then(data => {
+					if (data.length) {
+						return data[0];
+					} else {
+						throw new Error('We haven\'t got PLISTA!');
```

Comment:
what does it mean? why we haven't got it?
I'd focus this on "Plista returned no recomendations" or something like that